firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -307,7 +310,11 @@ private final void doRegistrationInternal(boolean forceRefresh) {
     }
 
     triggerOnStateReached(prefs);
+    networkExecutor.execute(() -> run(forceRefresh));
```

Comment:
just realized, we need to be a little careful, here.

If the subsequent call has a different value for "forceRefresh" then we need to be sure that it gets enqueued, otherwise the flag value can get lost.

Reply:
the force refresh values come from the main thread and will be added on to the network executor. I dont see how the flag gets lost/ overwritten.

@fredquintana wdyt can go wrong?